wiki/Tournaments/osu!_UCI/LANCross/en.md

```diff
@@ -0,0 +1,107 @@
+---
+tags:
+  - osu! UCI
+  - UCI
+  - Zotcade
+---
+
+# LANCross spring 2020 irvine culture
+
+![LANCross banner](img/banner.png)
+
+**LANCross spring 2020 irvine culture** is an osu! and osu!mania 4K BYOC LAN tournament hosted by [osu! UCI](/wiki/Organisations/osu!_UCI) in coordination with [The Association of Gamers at UC Irvine](https://www.facebook.com/TAGatUCI/). It will take place during *Zotcade 2020*, a gaming festival held at the University of California, Irvine (*UCI*). This will be osu! UCI's fifth installment of yearly LAN tournaments at Zotcade.
+
+## Event details
+
+![Map of the Student Center showing Moss Cove](/wiki/shared/UCI-Moss-Cove-map.png "Moss Cove in the Student Center")
+
+LANCross will take place on **Sunday, April 5, 2020** at the **UCI Student Center (311 W Peltason Dr, Irvine, CA 92697)**, in room **Moss Cove AB**. The room will be open from 9 a.m. to at least 9 p.m., and the tournaments are expected to start around 11 a.m. Participants should arrive as early as possible! The adjacent Student Center Parking Structure has parking priced around $10.
```

Comment:
it's $13, might as well be specific if it's known?

Reply:
imagine actually parking there and not at in n out for free